src/api/modelTypes.js

```diff
@@ -749,9 +749,42 @@ type MessageBase = $ReadOnly<{|
   content_type: 'text/html',
   // display_recipient handled on PmMessage and StreamMessage separately
 
-  // Optional because we only store it in Redux if it's coming from a server
-  // with FL >=118; see comment on MessageEdit.
-  edit_history: $ReadOnlyArray<MessageEdit> | null,
+  /**
+   * A possibly incomplete view of the message's edit history.
+   *
+   * Null if it's coming from a server with FL <118; see comment on
+   * MessageEdit.
```

Comment:
nit: "null" is a keyword, so keep it case-sensitive:

```suggestion
   * This is null if it's coming from a server with FL <118; see comment on
   * MessageEdit.
```